Topics/math-fundamentals.md

```diff
@@ -0,0 +1,172 @@
+
+Moreover, Virgilio's guide to Matrix Algebra is an _interactive Jupyter Notebook_, with exercises included!
+
+We suggest to go through this latter resource once (or if) you are already practicing with the [Python](https://www.python.org/) programming language. 
+
+#### Check your knowledge
+
+Test your knowledge with [**these**](https://web.pdx.edu/~erdman/LINALG/Linalg_pdf.pdf) exercises.
+
+Anyway you can find a ton of them online, and remember, that _the more you make exercise, the better!_
+
+### Optimization
+
+Calculus, Linear Algebra, and Matrix Algebra are the 3 fundamental pillars you need to be confident with in order to understand most of the mathematical implications of Machine Learning systems.
+
+This is important in Data Science because developing Machine Learning predictive models play a big role during a Data Science project, and these models are often the most useful outcome!
+
+The **optimization course** is a bit more advanced, so maybe you could not tackle _during the first pass of this guide_, but later be sure to take the following lectures, from the professor [Geoff Gordon](https://www.youtube.com/channel/UC4XBoaEXuvob0mQraQJ7wNw):
+
+[**Introduction to Convex Optimization**](https://www.youtube.com/playlist?list=PL7y-1rk2cCsDOv91McLOnV4kExFfTB7dU)
+
+Moreover, if you want to dive deeper (very deep), you can hold this close to your bed:
+
+[The Convex Optimization Book](https://web.stanford.edu/~boyd/cvxbook/bv_cvxbook.pdf) (advanced book.
+
+### Ask Questions
+A rule of thumb rule to learn fast and effectively is to ask questions and read other's questions and answers.
+
+Join communities of people interested in the topic (e.g. Reddit): here you can find discussions, search by keywords (e.g. "matrix multiplication"), and ask questions, with experts who will answer and help you.
+
+Some _tips_ regarding questions:
+
+- Try to form specific, well-written questions, to minimize the time used by the respondent.
+- Do not ask a question whose answer is found with a quick search on google.
+- If the questions are too general or show laziness they'll likely remain unanswered...
+
+Some subreddits you can subscribe to are:
+
+- [r/LearnMath](https://www.reddit.com/r/learnmath/)
+- [r/Math](https://www.reddit.com/r/math/)
+- [r/Calculus](https://www.reddit.com/r/calculus/)
+- [r/Optimization](https://www.reddit.com/r/optimization/)
+
+Two other good places to post (well structured) questions are:
+
+- [HackerNews](http://hn.premii.com/)
+- [Quora](https://www.quora.com/)
+
+### Conclusions
+
+If you follow the study plan of the guide, take the time to fully understand the proposed courses, and do exercises, you will be equipped with an excellent basic knowledge of mathematics that is needed to do projects of Data Science, and in particular, build predictive models of Machine Learning.
```

Comment:
So one slight nit-picking I will do, about this line:

> you will be equipped with an excellent basic knowledge of mathematics that is needed to do projects of Data Science, and in particular, build predictive models of Machine Learning.

Data Science requires a lot more than just maths, stats & probs, although this resource is good to help one thru a decent ML project, maybe even a DL/RL project.

The message is more for the original authors at MIT, not so much to you.

Reply:
okay.